src/APP/index.js

```diff
@@ -0,0 +1,7 @@
+var React = require('react');
```

Comment:
I suggest you to use
Import Component from 'path/to/component' sytanx instead instead of var Component = require('path/to/component')